package.json

```diff
@@ -39,7 +40,6 @@
     "semantic-release": "^17.2.3"
   },
   "peerDependencies": {
-    "eslint-plugin-react": "^7.16.0",
```

Comment:
this still does need to be a peer dep tho, otherwise your users might have a duplicate version of it